srml/staking/src/tests.rs

```diff
@@ -2651,7 +2647,7 @@ fn validator_payment_ratio_should_work() {
 			vec![validator_stash],
 		));
 
-		assert_eq!(Staking::reward_validator(&validator_stash, COIN).0.peek(), 0);
```

Comment:
Why this commented out?

Reply:
I commented out this becauce of the following errors:

```
error[E0599]: no method named `peek` found for type `(darwinia_balances::imbalances::PositiveImbalance<mock::Test>, (u128, std::vec::Vec<NominatorReward<u64, u128>>))` in the current scope
    --> srml/staking/src/tests.rs:2650:64
     |
2650 |         assert_eq!(Staking::reward_validator(&validator_stash, COIN).peek(), 0);
     |                                                                      ^^^^ method not found in `(darwinia_balances::imbalances::PositiveImbalance<mock::Test>, (u128, std::vec::Vec<NominatorReward<u64, u128>>))`
```
I found `peek` method in `darwinia_balances::imbalances::PositiveImbalance`. However, it doesn't work in `darwinia_balances::imbalances::PositiveImbalance<mock::Test>`. I don't know what's wrong with it and if I need to add something in `mock.rs`.

Reply:
The following way works:

```
assert_eq!(Staking::reward_validator(&validator_stash, COIN).0.peek(), 0);
```